var/spack/repos/builtin/packages/xsdktrilinos/package.py

```diff
@@ -65,7 +65,9 @@ def cmake_args(self):
         ])
 
         # Fortran lib
-        if spec.satisfies('%gcc') or spec.satisfies('%clang'):
+        if (spec.satisfies('%gcc') or
+                spec.satisfies('%clang') or
+                spec.satisfies('%apple-clang')):
             libgfortran = os.path.dirname(os.popen(
                 '%s --print-file-name libgfortran.a' %
                 join_path(mpi_bin, 'mpif90')).read())
```

Comment:
Ouch that's hacky -- people sometimes use xlf with clang. But that's not a part of this PR, maybe someone will go back and take care of it later.